src/createServer.js

```diff
@@ -1,3 +1,74 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const errorsTypes = {
+  notOriginalText: 'Text to convert is required.'
+  + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  notTargetCase: '"toCase" query param is required.'
+  + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  wrongTargetCase: 'This case is not supported.'
+  + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normaliseUrl = new URL(req.url, `http://${req.headers.host}`);
+    const { pathname, searchParams } = normaliseUrl;
+    const originalText = pathname.slice(1);
+    const params = new URLSearchParams(searchParams);
+    const targetCase = params.get('toCase');
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Let's move it outside this function
Would be better to move constants to separate file